setup.py

```diff
@@ -74,9 +74,11 @@ def build_extension(self, ext):
 # README file as long_description.
 long_description = open('README.md', encoding='utf-8').read()
 
+__version__ = '0.0.2'
```

Comment:
Will this behave correctly when pushing a new release? I thought v0.0.2 already existed.

Reply:
You are correct. When pushing a new release, this needs to be updated to 0.0.3. I changed it to 0.0.2 so that when a new version needs to be pushed it can be changed to 0.0.3.